publish/binaries/BUILD.bazel

```diff
@@ -32,6 +32,7 @@ BINARIES = {
     "certificate-issuer": "//rs/boundary_node/certificate_issuance/certificate_issuer:certificate-issuer",
     "certificate-syncer": "//rs/boundary_node/certificate_issuance/certificate_syncer:certificate-syncer",
     "denylist-updater": "//rs/boundary_node/denylist_updater:denylist-updater",
+    "e2e-test-driver": "//rs/scenario_tests:e2e-test-driver",
```

Comment:
please group together e2e-test-driver and ic-workload-generator and in a comment explain they are needed by the DRE to for release qualification.